site/docs/spark-ddl.md

```diff
@@ -182,6 +182,28 @@ ALTER TABLE prod.db.sample
 ADD COLUMN point.z double
 ```
 
+```sql
+-- create a nested array column of struct
+ALTER TABLE prod.db.sample
+ADD COLUMN points array<struct<x: double, y: double>>;
+
+-- add a field to the struct with in array
+ALTER TABLE prod.db.sample
+ADD COLUMN points.element.z double
```

Comment:
Nit / non-blocking: You might consider documenting that `element` is a keyword for referring to the elements of an array.

It's not necessarily clear to me as a reader that element is a keyword, and when it should be used. Possibly others feel differently though. The comment in the test is great in my opinion.

Reply:
done